Mindscape.Raygun4Net.NetCore.Common/RaygunClientBase.cs

```diff
@@ -327,9 +327,10 @@ public virtual async Task SendAsync(Exception exception, IList<string> tags, IDi
     /// </summary>
     /// <param name="raygunMessage">The RaygunMessage to send. This needs its OccurredOn property
     /// set to a valid DateTime and as much of the Details property as is available.</param>
-    public void SendInBackground(RaygunMessage raygunMessage)
+    public Task SendInBackground(RaygunMessage raygunMessage)
```

Comment:
```suggestion
        public Task SendInBackground(RaygunMessage raygunMessage)
    {
       return   SendInBackgroundButReturnCompletedTask(raygunMessage);
    }
    
    ///<summary>For backwards compatibility we need to return a Task. 
    /// But we DO NOT want to block/await on SendInBackground(). 
    /// Which is why we _ discard the Task.Run() and return a Task.CompletedTask
    ///</summary>
    private Task SendInBackgroundButReturnCompletedTask(RaygunMessage raygunMessage)
    {    
       _ =   Task.Run(() => Send(raygunMessage));
      return Task.CompletedTask;
    }
    
```

What we are doing here with Tasks looks 'weird'. 
But we have our reasons. 
Comment rot is a thing. 

So suggesting pulling this into a private method which is a bit more explicit

Reply:
I added comment but i applied it as a normal comment not an xml comment so we don't need to extract another method.